xds/src/main/java/io/grpc/xds/EnvoyProtoData.java

```diff
@@ -65,89 +48,11 @@
 // TODO(chengyuanzhang): put data types into smaller categories.
 final class EnvoyProtoData {
   static final String TRANSPORT_SOCKET_NAME_TLS = "envoy.transport_sockets.tls";
```

Comment:
@sanjaypujare This is a really bad thing. I don't know why it is declared here and used somewhere else. Am I the reviewer for this original change :P? I would have not approved such things in original PR.

Reply:
Okay it should have been defined in EnvoyServerProtoData. It could have been the IDE refactor. You can just move it to that class.

Reply:
I would just leave it in this PR. It's irrelevant to what I am changing. It was just a FYI. :slightly_smiling_face: